jellyfinclient/src/utilities/storage/store.js

```diff
@@ -18,21 +17,21 @@ const rootPersistConfig = {
 //TODO add bulk storage (for sync and download)
 
 const authPersistConfig = {
-    key: 'auth',
+    key: 'authCredentials',
     storage: sensitiveStorage,
     stateReconciler: hardSet,
 }
 
 const connectPersistConfig = {
-    key: 'connect',
+    key: 'connectionStatus',
```

Comment:
Shouldn't these keys mirror the keys in the combined reducer below?

Reply:
Yes I missed that. Updated in e095478e5d3b0fec561443ac2b070a3244a00e50